core/src/main/java/com/datastrato/gravitino/catalog/BaseCatalog.java

```diff
@@ -172,4 +178,12 @@ public Audit auditInfo() {
     Preconditions.checkArgument(entity != null, "entity is not set");
     return entity.auditInfo();
   }
+
+  protected CatalogOperations asProxyOps(CatalogOperations ops, ProxyPlugin plugin) {
+    return OperationsProxy.getProxy(ops, plugin);
+  }
+
+  protected ProxyPlugin newProxyPlugin(Map<String, String> config) {
+    return null;
```

Comment:
It is better using optional to avoid null check.

Reply:
Fixed.